Tiltfile

```diff
@@ -26,12 +26,11 @@ Here's a quick rundown of these services and their properties:
   * Other notes: Uses yarn. Does a `yarn install` for package dependencies, only if the dependencies have changed
 """
 
-def get_username():
-  return str(local('whoami')).rstrip('\n')
+username() = str(local('whoami')).rstrip('\n')
```

Comment:
What is this parenthesis syntax in the left hand side?

Reply:
it's a typo 😅

Reply:
wow wait why did this work??

Reply:
...okay it's heckin' weird that starlark allows that??

Reply:
Hmm it doesn't in the repl

```
Welcome to Starlark (go.starlark.net)
>>> foo() = "bar"
<stdin>:1:1: can't assign to callexpr
```

Reply:
wiiiild. wat.

Reply:
...
......
..........
i just don't think i ever ran `tilt up` after making this change cuz it def doesn't work DON'T LOOK AT MEEEEE